playGame.py

```diff
@@ -64,11 +64,11 @@ def start_game(self):
 
             if self.board.is_there_a_winner():
                 print('Congratulations ! You have won the game!')
-                break
+                #break
```

Comment:
Probably should just delete these rather than comment out. git has the history - you don't need to preserve it.